frontend/src/api/auth/getAuth.tsx

```diff
@@ -0,0 +1,30 @@
+import { useMutation, UseMutationResult } from 'react-query';
+import Cookies from 'js-cookie';
+
+import { apiClientWrapper } from 'api/base';
+import { API } from 'constants/routes';
+import { COOKIE_VALUES, REQUEST_ERRORS } from 'constants/authConstants';
+
+const useGetAuth = (): UseMutationResult =>
+  useMutation(async (initialData: string | unknown) => {
+    let tokenResponse: {
+      accessToken?: string;
+      _id?: string;
+      error?: unknown;
+    };
+    const apiClient = apiClientWrapper(COOKIE_VALUES?.uniqAccessToken);
+
+    try {
+      const response = await apiClient.post(API.getToken, initialData);
+      tokenResponse = response.data;
+      const resAccess = JSON.stringify(tokenResponse.accessToken);
+      const resUserId = JSON.stringify(tokenResponse._id);
```

Comment:
Rename please contants to something more clear, example: `accessToken` and `userId`

Reply:
Renamed